src/upkeeps/UpKeepManager.sol

```diff
@@ -0,0 +1,496 @@
+    /// @param _name member's name
+    /// @param _gasLimit amount of gas to provide the target contract when performing upkeep
+    /// @param _existingUpKeepId optional upKeepId to add member which has being registered from other admins
+    function addMember(address _memberAddress, string memory _name, uint256 _gasLimit, uint256 _existingUpKeepId)
+        external
+        onlyGovernance
+    {
+        /// @dev sanity checks before adding a new member in storage
+        if (_memberAddress == address(0)) revert ZeroAddress();
+        if (membersInfo[_memberAddress].gasLimit != 0) revert MemberAlreadyRegister(_memberAddress);
+        if (_gasLimit == 0) revert ZeroUintValue();
+        if (bytes(_name).length == 0) revert EmptyString();
+
+        _members.add(_memberAddress);
+        membersInfo[_memberAddress] = MemberInfo({
+            name: _name,
+            gasLimit: _gasLimit,
+            // NOTE: when `_existingUpKeepId` is greater than zero, assumes that other admin has registered
+            //       and there are not needs to register again
+            upKeepId: _existingUpKeepId > 0 ? _existingUpKeepId : _registerUpKeep(_memberAddress, _gasLimit, _name)
+        });
+
+        emit NewMember(_memberAddress, _name, _gasLimit, block.timestamp);
+    }
+
+    /// @dev Cancels an member's upkeep job
+    /// @notice only callable via governance
+    /// @param _memberAddress contract address to be cancel upkeep
+    function cancelMemberUpKeep(address _memberAddress) external onlyGovernance {
+        if (!_members.contains(_memberAddress)) revert NotMemberIncluded(_memberAddress);
+
+        // NOTE: only member which upkeep is being cancelled can be removed
+        uint256 upKeepId = membersInfo[_memberAddress].upKeepId;
+        CL_REGISTRY.cancelUpkeep(upKeepId);
+    }
+
+    /// @dev Withdraws LINK funds and remove member from manager
+    /// @notice only callable via governance
+    /// @param _memberAddress contract address to be remove from manager
+    function withdrawLinkFundsAndRemoveMember(address _memberAddress) external onlyGovernance {
+        if (!_members.contains(_memberAddress)) {
+            revert NotMemberIncluded(_memberAddress);
+        }
+
+        uint256 upKeepId = membersInfo[_memberAddress].upKeepId;
+        _members.remove(_memberAddress);
+        delete membersInfo[_memberAddress];
+
+        // NOTE: only member which upkeep is being cancelled can be removed
+        (,,,,,, uint64 maxValidBlocknumber,) = CL_REGISTRY.getUpkeep(upKeepId);
+        // https://etherscan.io/address/0x02777053d6764996e594c3e88af1d58d5363a2e6#code#F1#L738
+        if (maxValidBlocknumber == UINT64_MAX) revert UpKeepNotCancelled(upKeepId);
+
+        CL_REGISTRY.withdrawFunds(upKeepId, address(this));
+
+        emit RemoveMember(_memberAddress, upKeepId, block.timestamp);
+    }
+
+    /// @dev  Sweep the full LINK balance to techops
+    function sweepLinkFunds() external onlyGovernance {
```

Comment:
Sweep generalization

May as well allow to sweep all from governance to any recipient
Saves extra risks in the future as I don't believe this contract has a generic `call`

Reply:
see comment https://github.com/Badger-Finance/badger-avatars/pull/16#discussion_r1085246310

Reply:
added the generalised sweep erc20 at the end in 236d9ea3eaa0d12c55c30183fdc99a63b2ddf0e5